ivy_tests/test_docstrings.py

```diff
@@ -71,7 +71,13 @@ def test_docstrings(backend):
         if k == "Array":
             for method_name in dir(v):
                 method = getattr(ivy.Array, method_name)
-                if method_name in skip_arr_cont or helpers.docstring_examples_run(
+                try:
```

Comment:
you have only added this for `ivy.Array` instance method tests, we should also add it for `ivy.Container` instance method tests and functional tests below.